contrib/kind-helm.sh

```
@@ -257,6 +294,102 @@ parse_args() {
+            -ifa | --ipfix-cache-active-timeout ) shift
+                                                  OVN_IPFIX_CACHE_ACTIVE_TIMEOUT=$1
+                                                  ;;
+            -lcl | --libovsdb-client-logfile )    shift
+                                                  LIBOVSDB_CLIENT_LOGFILE=$1
+                                                  ;;
+            -eb | --egress-gw-separate-bridge )   OVN_SECOND_BRIDGE=true
+                                                  ;;
+            -lr | --local-kind-registry )         KIND_LOCAL_REGISTRY=true
+                                                  ;;
+            -ep | --experimental-provider )       shift
+                                                  KIND_EXPERIMENTAL_PROVIDER=$1
```

Shellcheck (0.11.0)

[warning] KIND_EXPERIMENTAL_PROVIDER appears unused. Verify use (or export if used externally).

(SC2034)